.github/workflows/build-mudlet-win.yml

```diff
@@ -47,6 +47,10 @@ jobs:
         $GITHUB_WORKSPACE/CI/setup-windows-sdk.sh
         $GITHUB_WORKSPACE/CI/validate-deployment-for-windows.sh
 
+    - name: Enable sentry.io for Windows releases
```

Comment:
There is probably a better way to incorporate this `WITH_SENTRY` value (`yes`) into the GH Actions "environment". I'd have to give it some thought as to what that would be...